tests/acceptance/bootstrap/CollaborationContext.php

```diff
@@ -323,4 +324,34 @@ public function userGetsTheInformationOfTheLastOpenedFileUsingWopiEndpoint(strin
 			)
 		);
 	}
+
+	/**
+	 * @Then /^the response should (not|)\s?contain following MIME Type:$/
+	 *
+	 * @param string    $shouldOrNot   (not|)
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theFollowingMimeTypesShouldExistForUser(string $shouldOrNot, TableNode $table):void {
+		$rows = $table->getRows();
+		foreach ($rows as $row) {
+			$responseArray = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse())['mime-types'];
+			$mimeTypes = \array_column($responseArray, 'mime_type');
+			if ($shouldOrNot === "not") {
+				Assert::assertFalse(
+					\in_array($row[0], $mimeTypes),
+					"the response should not contain the tag $row[0].\nResponse\n"
+					. print_r($responseArray, true)
```

Comment:
log the mimetypes not the whole response
```suggestion
					. $mimeTypes
```